client/src/js/services/FilterService.js

```diff
@@ -30,7 +30,7 @@ function FilterService(Store) {
   //       with a toggle between the array to populate and the default value
   FilterList.prototype.registerDefaultFilters = function registerDefaultFilters(filterDefinitions) {
     var formattedFilters = filterDefinitions.map(function (filterDefinition) {
-      var filter = new Filter(filterDefinition.key, filterDefinition.label, filterDefinition.valueFilter);
+      var filter = new Filter(filterDefinition.key, filterDefinition.label, filterDefinition.valueFilter, filterDefinition.comparitor);
```

Comment:
Yay!  @mbayopanda and I had a discussion about this ... really great to see it fixed!